frappe/public/js/frappe/form/controls/markdown_editor.js

```diff
@@ -0,0 +1,55 @@
+frappe.ui.form.ControlMarkdownEditor = frappe.ui.form.ControlCode.extend({
+	make_input() {
+		this._super()
+		this.$input.height(300)
+		this.text_area = this.$input.first()
+		this.showing_preview = false
+
+		this.make_preview_container()
+		this.make_preview_button()
+	},
+
+	set_disp_area(value) {
+		value = value || ""
+		this.value = frappe.markdown(value)
+		this._super()
```

Comment:
Check how does it look when the Form is in submitted state

Reply:
Okay. Wait.

Reply:
Looks Okay. Need Screenshot?